### PR TITLE
Adding a new subchapter on rdm toolkits

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* Adding a new subchapter on rdm toolkits, rdm-toolkits.md

### List of changes proposed in this PR (pull-request)

* News subchapter for rdm

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/reproducible-research/rdm/rdm-toolkits.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+### Why should you use a research data management toolkit?
+
+A research data management toolkit will help you to maximize the efficiency and integrity of your research, and to increase your visibility and impact. Consulting a research data management toolkit will also learn you how to comply with requirements from funders, publishers and others. Last but not least, the toolkit can assist in finding and re-using datasets for your own research.
+
+
+### How to choose a research data management toolkit?
+
+Information and materials are often sorted by topic and audience. Materials can be found relevant to your role (researcher, research support, IT specialist), your research domain (e.g.for life sciences: plant phenotyping, human data,...) or your problem (e.g. storage, data transfer).
```
Comment (kderuyck) [inline book/website/reproducible-research/rdm/rdm-toolkits.md:18]:
```suggestion
Information and materials are often sorted by topic and audience. Materials can be found relevant to your role (researcher, research support, IT specialist), your research domain (for life sciences: plant phenotyping, human data,...) or your problem (such as storage, data transfer).  
```

**Review by ahornos — APPROVED** [review]

File: book/website/_toc.yml [hunk 2]
```diff
@@ -93,6 +93,8 @@
       file: reproducible-research/rdm/rdm-metadata
     - title: Sharing and Archiving Data
       file: reproducible-research/rdm/rdm-sharing
+    - title: RDM Toolkits
```
Comment (kderuyck) [inline book/website/_toc.yml:96]:
```suggestion
    - title: Research data management toolkits
```

File: book/website/reproducible-research/rdm/rdm-toolkits.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+A wide variety of research data management toolkits are available. Here, we list a number of toolkits:
+
+* [JISC research data management toolkit](https://rdmtoolkit.jisc.ac.uk/rdm-for-researchers/)
+* [ELIXIR research data management toolkit](https://rdm.elixir-europe.org/index.html)
+* [University of New Hampshire data management toolkit](https://libraryguides.unh.edu/datamanagement)
+
+
+Notes
+*Many more toolkits are available: lots of them are however rather guides than toolkits. One could argue about the definition of a RDM toolkit*
```
Comment (kderuyck) [inline book/website/reproducible-research/rdm/rdm-toolkits.md:29]:
```suggestion
*Many more toolkits are available. Lots of them are rather guides than toolkits. One could argue about the definition of a RDM toolkit*
```

**Comment by malvikasharan** [comment]
@all-contributors please add @kderuyck for content

**Pull request merged**